django/db/models/manager.py

```diff
@@ -87,8 +86,7 @@ def manager_method(self, *args, **kwargs):
 
         new_methods = {}
         # Refs http://bugs.python.org/issue1785.
```

Comment:
I think it's related to the `predicate = inspect.isfunction if six.PY3 else inspect.ismethod` line and can be removed (31fadc120213284da76801cc7bc56e9f32d7281b).